<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="windowTitle">
    <string>Form</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="pointView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="lineView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDraw"/>
   <addaction name="actionRemove"/>
   <addaction name="actionSolve"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
  </widget>
  <action name="actionDraw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/oxygen-icons/draw-freehand.png</normaloff>:/icons/oxygen-icons/draw-freehand.png</iconset>
   </property>
   <property name="text">
    <string>Draw line</string>
   </property>
   <property name="toolTip">
    <string>Draw a line</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/oxygen-icons/draw-eraser.png</normaloff>:/icons/oxygen-icons/draw-eraser.png</iconset>
   </property>
   <property name="text">
    <string>Remove line</string>
   </property>
   <property name="toolTip">
    <string>Remove a line or point</string>
   </property>
  </action>
  <action name="actionSolve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/oxygen-icons/media-playback-start.png</normaloff>:/icons/oxygen-icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Solve</string>
   </property>
   <property name="toolTip">
    <string>Solve hitofude gaki</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/oxygen-icons/edit-clear.png</normaloff>:/icons/oxygen-icons/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
